src/components/dateCourse/timeline.tsx

```
@@ -10,45 +10,52 @@ import Cash from '@/assets/icons/cash_Blank.svg?react';
 import CheckSuccess from '@/assets/icons/Check-Success-Blank.svg?react';
 import Location from '@/assets/icons/Location_Blank.svg?react';
 
-function Timeline({ end = false, title, time, address, price, tags, menu }: TTimeline) {
+function Timeline({ end = false, image, name, placeCategoryResponseList, roadNameAddress, averagePrice, time, signatureDish }: TTimeline) {
     const [open, setOpen] = useState(false);
     return (
         <div className="flex w-full gap-2 flex-col">
             <div className={`${!end && 'hover:cursor-pointer'} flex justify-center gap-2 items-center`} onClick={() => (end ? undefined : setOpen(!open))}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)